app/scripts/views/complete_sign_up.js

```diff
@@ -94,9 +96,11 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
               return false;
             }
 
-            return self.getAccount().isSignedIn()
+            var account = self.getAccount();
+            return account.isSignedIn()
               .then(function (isSignedIn) {
                 if (isSignedIn) {
+                  self.interTabSend(EVENTS.SIGNIN_SUCCESS, account);
```

Comment:
> ! self.relier.isDirectAccess()

I don't know because I'm unsure what `! self.relier.isDirectAccess()` means. I thought it meant that the user will not be signed in, was that wrong?